starter-code/pizza.js

```diff
@@ -1 +1,72 @@
 // Write your Pizza Builder JavaScript in this file.
+var $pepperonniButton = $('.btn-pepperonni');
+var $mushroomButton = $('.btn-mushrooms');
+var $greenPepperButton = $('.btn-green-peppers');
+var $sauceButton = $('.btn-sauce');
+var $crustButton = $('.btn-crust');
+
+var $pepperonni = $('.pep');
+var $mushroom = $('.mushroom');
+var $greenPepper = $('.green-pepper');
+var $glutenFreeCrust = $('.crust-gluten-free');
+var $normalCrust = $('.crust');
+var $typeOfSauce = $('.sauce')
+var $activate = $('.active');
+
+var $pizzaPrices = $('.price ul li');
+
+$pepperonniButton.click(function() {
+    $pepperonni.toggle();
+    $($activate[0]).toggleClass("active", 'btn btn-pepperonni');
+    $($pizzaPrices[0]).toggle()
+
+});
+$mushroomButton.click(function() {
+    $mushroom.toggle();
+    $($activate[1]).toggleClass("active", 'btn btn-mushroom');
+    $($pizzaPrices[1]).toggle()
+
+
+});
+$greenPepperButton.click(function() {
```

Comment:
Do you see how you repeat the same code again and again. Maybe you can abstract that into a function.